templates/compute_pulse_heights.ipynb

```diff
@@ -2,139 +2,37 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    bin_factor = 100
</pre><p>bin_factor and x_cutoff can probably just be wrapped in as defaults and don't need to be defined here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>